### PR TITLE
feat: extract logger to a separate schema component

### PR DESCRIPTION
**Pull request by ronenkapelian**
Logger should also be defined as seperated and independant schema so the boilerplate schema will refer to it and also other schemas

### CODE REVIEW

**Review by netanelC — APPROVED** [review]

File: schemas/common/telemetry/logger/v1.schema.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
+{
+  "$id": "https://mapcolonies.com/common/telemetry/logger/v1",
+  "title": "commonTelemetryLoggerV1",
```
Comment (CptSchnitz) [inline schemas/common/telemetry/logger/v1.schema.json:3]: no need for common in the title

File: schemas/common/telemetry/logger/v1.schema.json [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+      "default": "info",
+      "description": "The log level",
+      "x-env-value": "LOG_LEVEL",
+      "enum": ["trace", "debug", "info", "warn", "error", "fatal"]
+    },
+    "prettyPrint": {
+      "default": false,
+      "description": "Whether to pretty print logs",
+      "type": "boolean",
+      "x-env-value": "LOG_PRETTY_PRINT"
```
Comment (CptSchnitz) [inline schemas/common/telemetry/logger/v1.schema.json:33]:
```suggestion
      "x-env-value": "LOG_PRETTY_PRINT_ENABLED"
```
so we dont break API

**Review by CptSchnitz — CHANGES_REQUESTED** [review]

File: schemas/common/telemetry/logger/v1.schema.json (hunk 1 above)
Comment (ronenkapelian) [reply schemas/common/telemetry/logger/v1.schema.json:3]: but other examples AKA base and tracing include "common" in the title so I did it unified also in logger

File: schemas/common/telemetry/logger/v1.schema.json (hunk 2 above)
Comment (ronenkapelian) [reply schemas/common/telemetry/logger/v1.schema.json]: done

File: schemas/common/telemetry/logger/v1.schema.json (hunk 1 above)
Comment (ronenkapelian) [reply schemas/common/telemetry/logger/v1.schema.json]: done

**Review by CptSchnitz — APPROVED** [review]

**Pull request merged**